hwangsae/recorder.h

```diff
@@ -26,6 +26,7 @@
 #include <glib-object.h>
 
 #include "types.h"
+#include "common.h"
```

Comment:
Why include common in `*.h` when it's used only internally in the recorder's implementation? Why not put it into `recorder.c`?

Reply:
Same reason that above, to allow `HwangsaeHttpServer` to use `hwangsae_common_get_local_ip` to form the URL. As before, I am open to change it if you think is better in a different way.

Reply:
Please clarify, the approach you prefer to use.

Reply:
This means we need to install `common.h` which is internally used. We should keep public APIs and headers simple.